packages/solid-query/src/createQueries.ts

```diff
@@ -150,18 +150,18 @@ export type QueriesResults<
 export function createQueries<T extends any[]>(
   queriesOptions: () => {
     queries: readonly [...QueriesOptions<T>]
-    queryClient?: QueryClient
   },
+  queryClient?: () => QueryClient,
```

Comment:
```suggestion
  queryClient?: Accessor<QueryClient>,
```
Just a small change 😄